assets/src/edit-story/components/thumbnail/stories/demoThumbnails.js

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { THUMBNAIL_TYPES, THUMBNAIL_DIMENSIONS } from '..';
+import PagePreview from '../../carousel/pagepreview';
+
+const LayerIconImageOutput = (
```

Comment:
Everything here is just direct output from the `LayerIcon` component used in the Layer Panel